tests/acceptance/EditSignUpConfirmationEmailCest.php

```diff
@@ -3,21 +3,25 @@
 namespace MailPoet\Test\Acceptance;
 
 use Codeception\Util\Locator;
+use MailPoet\Test\DataFactories\Settings;
+
+require_once __DIR__ . '/../DataFactories/Settings.php';
 
 class EditSignUpConfirmationEmailCest {
 
   function edit(\AcceptanceTester $I) {
     $I->wantTo('Edit sign up confirmation email');
 
+    // make sure sign up confirmation is enabled
+    $settings = new Settings();
+    $settings
+      ->withConfirmationEmailEnabled();
+
     $I->login();
     $I->amOnMailPoetPage('Settings');
     $I->click('[data-automation-id="signup_settings_tab"]');
     $I->waitForText('Enable sign-up confirmation');
 
-    // make sure sign up confirmation is enabled
-    $I->click('[data-automation-id="enable_signup_confirmation"]');
-    $I->acceptPopup();
-
     // edit confirmation email
     $I->fillField('[data-automation-id="signup_confirmation_email_from_name"]', 'Confirmation Test From');
     $I->fillField('[data-automation-id="signup_confirmation_email_from_email"]', 'from-confirmation-test@example.com');
```

Comment:
methods `withConfirmationEmailSubject` and `withConfirmationEmailBody` should be used to replace the lines below. No need to fill inputs and submit the form.